src/layouting/layouting.jl

```diff
@@ -287,14 +261,6 @@ function padded_vcat(arrs::AbstractVector{T}, fillvalue) where T <: AbstractVect
     arr
 end
 
-function alignment2num(x::Symbol)
-    (x === :center) && return 0.5f0
-    (x in (:left, :bottom)) && return 0.0f0
-    (x in (:right, :top)) && return 1.0f0
-    return 0.0f0 # 0 default, or better to error?
-end
```

Comment:
I added `halign2num` and `valign2num` instead of this. Those functions also accept float values and you can optionally add an error message for when the conversion fails. I could also go back and extend this function instead, if that's preferred.